components/rmrk/Profile/ProfileDetail.vue

```diff
@@ -19,12 +19,9 @@
           </a>
         </h1>
 
-        <span v-if="!displayName">
-          Add on-chain recognition for
-          <nuxt-link :to="`/identity`">
-            {{ shortendId }}
-          </nuxt-link>
-        </span>
+        <nuxt-link v-if="!displayName && isMyProfile" :to="`/identity`">
```

Comment:
should work without templateString, no?
```suggestion
        <nuxt-link v-if="!displayName && isMyProfile" to="/identity">
```